integrations/launchdarkly/main.py

```diff
@@ -21,6 +21,20 @@ async def enrich_resource_with_project(endpoint: str, kind: str) -> dict[str, An
     return response
 
 
+async def enrich_feature_flag_with_status(
+    feature_flag: dict[str, Any]
+) -> dict[str, Any]:
+    projectKey = feature_flag["__projectKey"]
+    featureFlagKey = feature_flag["key"]
+
+    launchdarkly_client = initialize_client()
+    feature_flag_status = await launchdarkly_client.get_feature_flag_status(
+        projectKey, featureFlagKey
+    )
+    feature_flag.update({"__status": feature_flag_status})
+    return feature_flag
```

Comment:
how it possibility working? you are not ingesting a feature flag status
you are ingesting feature flag with a status